Resources/views/Gallery/index.html.twig

```diff
@@ -9,7 +9,7 @@ file that was distributed with this source code.
 
 #}
 
-{% sonata_template_box 'This is the gallery index template. Feel free to override it.' %}
+{% sonata_template_box 'This is the gallery index template. Feel free to override it.' 'SonataMediaBundle'%}
```

Comment:
After the fix in the core bundle. Do we still need this fix?

IMHO we should stop calling the translator. This message is for development only, so no reason for a translation.